sdk/schemaregistry/schema-registry-avro/test/utils/mockedRegistryClient.ts

```diff
@@ -50,8 +51,8 @@ export function createTestRegistry(neverLive = false): SchemaRegistry {
     return result!.properties;
 
     function newId(): string {
-      if (idCounter === testSchemaIds.length) {
-        throw new Error("Out of IDs. Generate more GUIDs and paste them above.");
+      if (idCounter >= testSchemaIds.length) {
+        return uuid();
```

Comment:
I guess I'm wondering a bit why we have any pre-generated GUIDs in the tests unless we rely on specific GUIDs.

Reply:
This was me and I forget my reason. LOL. I would agree that if it's ok to generate some it should be ok to generate all. I wouldn't keep a mix.

Reply:
It is my understanding that the hard coded ones are meant to serve as already registered schemas so some tests can call decode without having to call encode first/register schema explicitly first. I feel like it is reasonable to keep the hard-coded one based on this though it could make reading the tests a bit harder.

I can look into refactoring this in another PR.

Reply:
Ah, yes, that seems to ring a bell.